partcad/src/partcad/part_factory_step.py

```
@@ -136,22 +139,25 @@
-                for i in range(reader.NbShapes()):
-                    occ_shape = reader.Shape(i + 1)
-                    builder.Add(compound, occ_shape)
-                shape = compound
+                with pc_tracer.start_as_current_span(
+                    "*PartFactoryStep.instantiate.{OCP.STEPControl.STEPControl_Reader}"
+                ):
+                    reader = STEPControl_Reader()
+
+                    readStatus = reader.ReadFile(self.path)
+                    if readStatus != OCP.IFSelect.IFSelect_RetDone:
+                        raise ValueError("STEP File could not be loaded")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)